is_all_unique_elements.js

```diff
@@ -0,0 +1,14 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
Clear use of variables, well done. Have you explored other solutions for finding uniqueness? This solution has a nested loop - could be costly. What about using a map instead?